theme/themes/pastanaga/extras/main.less

```diff
@@ -650,6 +650,12 @@ img.responsive {
   height: auto;
 }
 
+#main .summary-listing-item-header {
+  color: @summaryItemTitle;
```

Comment:
See if it's possible to use some form of less inheritance for this selector (inheriting from the original h3). https://coderwall.com/p/4dwk_q/inheritance-in-less

Reply:
I have tried extending the `.summary-listing-item-header` selector to match the original `h3` styling however I found that unless I specifically define `h3` styling in a .less file I would not be able to extend the `.summary-listing-item-header` styling.

In my case, if I were to extend `.summary-listing-item-header` with predefined `h3` styling, like in the following code, `&:extend` works as expected.
```
h3 {
  color: @summaryItemTitle;
  font-size: 1.25em;
  font-weight: 500;
}

#main {
  .summary-listing-item-header {
    &:extend(h3 all);
  }
}
```
If I would delete the hard coded `h3` style  then `.summary-listing-item-header` would not inherit any styling from the original `h3`.


Should I make any changes?

Reply:
https://github.com/Semantic-Org/Semantic-UI-LESS/blob/806bd67fcc445ce254a55a7dd153dae408f4485d/definitions/globals/site.less#L55

the h3 is defined in semantic. Can you try to identify the problem?

Reply:
Hello, the changes were added via commit f58e96a. It seems that  `.summary-listing-item-header ` would not inherit the `<a>` color and I have to override it  on line [659](https://github.com/plone/volto/blob/f58e96aae947a1af2efb1b597e909fe07affb7b0/theme/themes/pastanaga/extras/main.less#L659)